File: src/com/artalgame/mobilerepetitor/Data/SubjectHelper.cs
using System;
using MR.Core.TestEntities;
using Android.Content;

namespace MR.Android.Data
{
	public static class SubjectHelper
	{
		public static string GetSubjectName (SubjectsEnumeration subjectType, Context context)
		{
			string subjectName = null;
			switch (subjectType) {
			case SubjectsEnumeration.Math:
				subjectName = context.GetString (com.flaxtreme.CT.Resource.String.Math);
				break;
			}
			return subjectName;
		}
	}
}

File: src/com/artalgame/mobile_repetitor/data/SubjectHelper.java
package com.artalgame.mobile_repetitor.data;

import android.content.Context;

import com.artal.mobile_repetitor.R;
import com.artal.mobile_repetitor.testEntities.SubjectsEnumeration;

/**
 * Use this class to apply simple operations with subjects.
 * @author Alexander Korolchuk
 *
 */
public class SubjectHelper
{
	public static String getSubjectName (SubjectsEnumeration subjectType, Context context)
	{
		String subjectName = null;
		switch (subjectType) {
		case Math:
			subjectName = context.getString(R.string.Math);
			break;
		}
		return subjectName;
	}
}